app.js

```diff
@@ -257,4 +365,10 @@ var pictureThree = document.getElementById('image-three');
 pictureOne.addEventListener('click', countClicks);
 pictureTwo.addEventListener('click', countClicks);
 pictureThree.addEventListener('click', countClicks);
+
+try {
+  pictureArray = JSON.parse(localStorage.pictureArray);
+} catch(error){
+  console.log('error');
+}
```

Comment:
Consider introducing this at the beginning of your app where you create picture array.  Instead of instantiating your pictureArray by creating a number of objects will which be overwritten before being used, you can make a choice of either load array from local storage if it exists or build your pictureArray with image objects if local storage is empty.